test/engine/SpatialJoinAlgorithmsTest.cpp

```diff
@@ -310,17 +321,17 @@ std::vector<std::string> mergeToRow(std::vector<std::string> part1,
   return result;
 };
 
-std::vector<std::vector<std::string>> unordered_rows{
-    std::vector<std::string>{"\"Uni Freiburg TF\"", "<node_1>", "<geometry1>",
-                             "POINT(7.835050 48.012670)"},
-    std::vector<std::string>{"\"Minster Freiburg\"", "<node_2>", "<geometry2>",
-                             "POINT(7.852980 47.995570)"},
-    std::vector<std::string>{"\"London Eye\"", "<node_3>", "<geometry3>",
-                             "POINT(-0.119570 51.503330)"},
-    std::vector<std::string>{"\"Statue of liberty\"", "<node_4>", "<geometry4>",
-                             "POINT(-74.044540 40.689250)"},
-    std::vector<std::string>{"\"eiffel tower\"", "<node_5>", "<geometry5>",
-                             "POINT(2.294510 48.858250)"},
+Rows unordered_rows{
+    Row{"\"Uni Freiburg TF\"", "<node_1>", "<geometry1>",
+        "POINT(7.835050 48.012670)"},
+    Row{"\"Minster Freiburg\"", "<node_2>", "<geometry2>",
+        "POINT(7.852980 47.995570)"},
+    Row{"\"London Eye\"", "<node_3>", "<geometry3>",
+        "POINT(-0.119570 51.503330)"},
+    Row{"\"Statue of liberty\"", "<node_4>", "<geometry4>",
+        "POINT(-74.044540 40.689250)"},
+    Row{"\"eiffel tower\"", "<node_5>", "<geometry5>",
```

Comment:
I think you don't even need the explicit `Row` for each entry, simnply making `{{...}, {...}}` should also work.